<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sauger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Status</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="7" column="2" colspan="2">
          <widget class="QLabel" name="tme_sidebrush">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="fanspeedLabel">
           <property name="text">
            <string>Fanspeed:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="mainBrushLabel">
           <property name="text">
            <string>Main Brush:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QProgressBar" name="con_filter">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QProgressBar" name="con_mainbrush">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="8" column="2" colspan="2">
          <widget class="QLabel" name="tme_filter">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="cleanedAreaLabel">
           <property name="text">
            <string>Cleaned Area:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="BatteryLabel">
           <property name="text">
            <string>Battery:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="sensorLabel">
           <property name="text">
            <string>Sensor Dirty:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="cleaningSinceLabel">
           <property name="text">
            <string>Cleaning Duration:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="nfo_fanspeed">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="sideBrushLabel">
           <property name="text">
            <string>Side Brush:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QProgressBar" name="con_sidebrush">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="2">
          <widget class="QLabel" name="tme_mainbrush">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="nfo_battery">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QProgressBar" name="con_sensor">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="filterLabel">
           <property name="text">
            <string>Filter:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="stateLabel">
           <property name="text">
            <string>State:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="2" colspan="2">
          <widget class="QLabel" name="tme_sensor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="4">
          <widget class="QPushButton" name="pushButtonRstSensor">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="tme_now">
           <property name="text">
            <string>HH:MM</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="2">
          <widget class="QLabel" name="nfo_clean">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QLabel" name="nfo_area">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="nfo_state">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QPushButton" name="pushButtonRstFilter">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QPushButton" name="pushButtonRstSide">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QPushButton" name="pushButtonRstMain">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="5">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxAction">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonPause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonHome">
         <property name="text">
          <string>Stop + Home</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonFind">
         <property name="text">
          <string>Find Beep</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Set Fanspeed:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="sliderFanSpeed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
